diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -373,6 +373,14 @@ namespace diff_drive_controller{
   {
     if (isRunning())
     {
+      // check that we don't have multiple publishers on the command topic
+      if (sub_command_.getNumPublishers() > 1)
+      {
+        ROS_ERROR_NAMED(name_, "Detected multiple publishers for diff_drive_controller. Going to brake.");
```

Comment:
I think this should be throttled, maybe at 1s period.

I'd also make the message a little more informative, but this is up to you.

`ROS_ERROR_STREAM_THROTTLE_NAMED(1.0, name_, "Detected " << sub_command_.getNumPublishers() << " publishers. Only 1 publisher is allowed. Going to brake.");`